velox/docs/functions/presto/decimal.rst

```diff
@@ -150,8 +183,10 @@ Decimal Functions
         p = min(38, p1 + s2 + max(0, s2 - s1))
         s = max(s1, s2)
 
-    Throws if y is zero or result cannot be represented using precision calculated
+    Throws if:
+    1. y is zero or result cannot be represented using precision calculated
     above.
+    2. :code:`result precision=max(s1, s2) - s1 + s2 exceeds 38`.
```

Comment:
This is confusing since "result precision" uses a different formula.

Reply:
This must be rescale factor.